### DISCUSSION TITLE
Changing displayed columns in Detail List Search Results

### DISCUSSION THREAD

**Discussion by BrainsWithBrawn** [microsoft-search/pnp-modern-search#1887] [Q&A]
Does anyone have a good step-by-step guide or walkthrough to customising the search results to display custom columns instead of the standard "Name, Created, Summary" array?

I used to be able to do this with a custom layout template in SharePoint 2013. the options available inthe Search Results webpart suggest there is a way to do it with the PNP webparts if I only knew how.

**Comment by wobba** [comment]
You can take a look at scenario docs https://microsoft-search.github.io/pnp-modern-search/scenarios/ for custom templates. The details list allows you to add more columns as well as described at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/#details-list

**Comment by PeterBehler** [comment]
Hi, It took me a couple tries the first time. In my case here is what I did and it finally worked.  I had a couple issues where my changes would get reset and I'm not sure why.
1st panel (Layout slots) - updated the slots to include the managed columns I used from the SP2013 layout like RefinableString06-09 and gave them slot names like Contact and Category.
1st panel (Selected properties) - I added the managed columns I referenced in the Layout slots.
2nd panel (Available layouts) - Selected the Details List layout
2nd page (Manage columns) - Updated the columns I wanted using the handlebar method {{slot item @root.slots.Contact}} and {{slot item @root.slots.Category}} and the Display name would be the viewer friendly version.

Save it enough times and retry it enough times and it finally stuck.  I'm not trying to figure if and how to I create a custom template and swap values with icons or add a hover card.  The one sample to create a custom template is to change the header text.
